external/fv3fit/fv3fit/pytorch/GraphHyperparameters.yml

```diff
@@ -0,0 +1,23 @@
+- name: GraphHyperparameters
+  config:
+    random_seed: 0
+    model_type: graph
+    hyperparameters:
+      loss:
+        loss_type: mse
+      graph_network:
+        in_feats:
```

Comment:
Please use a more verbose name, is this `input_features`? Shouldn't this value along with output_features be possible to determine algorithmically based on the input and output variable names, by looking at their sizes in the tfdataset?